GitCommands/Git/GitGpgController.cs

```diff
@@ -232,13 +232,13 @@ public string GetCommitVerificationMessage(GitRevision revision)
                 return null;
             }
 
-            var module = GetModule();
+            // '--raw' returns info in stderr
```

Comment:
Raw has nothing to do with this problem.  Pay attention to when raw is true.

Reply:
![image](https://user-images.githubusercontent.com/3629489/196004704-280f89b8-efbe-4ad5-829d-4c77eb3ce66c.png)

Reply:
See the Git man page. --raw prints the info on stderr in full format.

Reply:
And non raw

![image](https://user-images.githubusercontent.com/3629489/196004944-57d40bd2-5321-487c-960a-235c67d72dca.png)

My point is raw doesn't matter. 

![image](https://user-images.githubusercontent.com/3629489/196004976-38c1af14-bf04-434f-9cfc-096c9c925682.png)

Reply:
Moved to #10262